### ISSUE TITLE
Bug in 2.4.0 when conditions.json-file is empty

### ISSUE THREAD

**Issue by bive242** [Dom2364/PlaneAlerter#13]
Version 2.4.0 seem to have a bug that triggers when the conditions.json-file is empty (like it is when you run it for the first time in a new directory).

It can be avoided by using a conditions.json from version 2.3.0 (don't know if there are any differences between the two versions in this specific file).

This is the error I get when the conditions.json is empty (sorry for localized error information, but I think it's useful anyway):
`````
Information om att aktivera JIT-felsökning i stället för den 
här dialogrutan finns i slutet av det här meddelandet.

************** Undantagstext **************
System.ArgumentOutOfRangeException: Det angivna argumentet hamnade utanför intervallet med giltiga värden.
Parameternamn: index
   vid System.Windows.Forms.TreeNodeCollection.get_Item(Int32 index)
   vid PlaneAlerter.ConditionEditor.updateConditionList()
   vid PlaneAlerter.ConditionEditor..ctor()
   vid PlaneAlerter.PlaneAlerter.startConditionEditorToolStripMenuItem_Click(Object sender, EventArgs e)
   vid System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   vid System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   vid System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   vid System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   vid System.Windows.Forms.ToolStripItem.FireEventInteractive(EventArgs e, ToolStripItemEventType met)
   vid System.Windows.Forms.ToolStripItem.FireEvent(EventArgs e, ToolStripItemEventType met)
   vid System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   vid System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   vid System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   vid System.Windows.Forms.Control.WndProc(Message& m)
   vid System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   vid System.Windows.Forms.ToolStrip.WndProc(Message& m)
   vid System.Windows.Forms.ToolStripDropDown.WndProc(Message& m)
   vid System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   vid System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   vid System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Inlästa sammansättningar **************
mscorlib
    Sammansättningsversion: 4.0.0.0
    Win32-version: 4.8.4470.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework/v4.0.30319/mscorlib.dll
----------------------------------------
PlaneAlerter
    Sammansättningsversion: 2.4.0.0
    Win32-version: 2.4.0.0
    CodeBase: file:///C:/Users/pb19873/Downloads/PlaneAlerter_2.4.0/PlaneAlerter.exe
----------------------------------------
System.Windows.Forms
    Sammansättningsversion: 4.0.0.0
    Win32-version: 4.8.4400.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Sammansättningsversion: 4.0.0.0
    Win32-version: 4.8.4360.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Sammansättningsversion: 4.0.0.0
    Win32-version: 4.8.4390.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Configuration
    Sammansättningsversion: 4.0.0.0
    Win32-version: 4.8.4190.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Core
    Sammansättningsversion: 4.0.0.0
    Win32-version: 4.8.4470.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Xml
    Sammansättningsversion: 4.0.0.0
    Win32-version: 4.8.4084.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
System.Windows.Forms.resources
    Sammansättningsversion: 4.0.0.0
    Win32-version: 4.8.4084.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms.resources/v4.0_4.0.0.0_sv_b77a5c561934e089/System.Windows.Forms.resources.dll
----------------------------------------
Newtonsoft.Json
    Sammansättningsversion: 13.0.0.0
    Win32-version: 13.0.1.25517
    CodeBase: file:///C:/Users/pb19873/Downloads/PlaneAlerter_2.4.0/lib/Newtonsoft.Json.DLL
----------------------------------------
System.Runtime.Serialization
    Sammansättningsversion: 4.0.0.0
    Win32-version: 4.8.4455.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Runtime.Serialization/v4.0_4.0.0.0__b77a5c561934e089/System.Runtime.Serialization.dll
----------------------------------------
System.Numerics
    Sammansättningsversion: 4.0.0.0
    Win32-version: 4.8.4084.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Numerics/v4.0_4.0.0.0__b77a5c561934e089/System.Numerics.dll
----------------------------------------
System.Data
    Sammansättningsversion: 4.0.0.0
    Win32-version: 4.8.4455.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_32/System.Data/v4.0_4.0.0.0__b77a5c561934e089/System.Data.dll
----------------------------------------
WindowsBase
    Sammansättningsversion: 4.0.0.0
    Win32-version: 4.8.4470.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/WindowsBase/v4.0_4.0.0.0__31bf3856ad364e35/WindowsBase.dll
----------------------------------------
mscorlib.resources
    Sammansättningsversion: 4.0.0.0
    Win32-version: 4.8.4084.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/mscorlib.resources/v4.0_4.0.0.0_sv_b77a5c561934e089/mscorlib.resources.dll
----------------------------------------
`````

**Comment by Dom2364** [comment]
Just fixed the issue and re-released 2.4.0, thanks for the quick bug report!

**Comment by bive242** [comment]
No problems :)
The other bugs in 2.3.0 seem to be gone, so thanks for the quick work!
